src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2380,19 +2386,19 @@ When more than one generic information is specified, then a member can interact
 ----
 permission org.ow2.proactive.scheduler.permissions.ConnectToResourceManagerPermission;
 ----
-Administrative permission that allows linking the scheduler to an existing resource manager.
+_Administrative_ permission that allows linking the scheduler to an existing resource manager.
 
 ----
 permission org.ow2.proactive.scheduler.permissions.ChangePolicyPermission;
 ----
-Administrative permission that allows changing the server's <<_scheduling_policies,Scheduling Policy>>.
+_Administrative_ permission that allows changing the server's <<_scheduling_policies,Scheduling Policy>>.
```

Comment:
changing the server's .. -> changing the scheduler's
We need to use the same terminology in the documentation (either scheduler or server)

Reply:
I will write like you say